packages/framework/src/Form/Admin/Country/CountryFormType.php

```diff
@@ -119,18 +119,21 @@ public function configureOptions(OptionsResolver $resolver): void
     }
 
     /**
-     * @param $countryCodeValue
+     * @param mixed $countryCodeValue
```

Comment:
this was changed to string during [implementation of phpstan](https://github.com/shopsys/shopsys/pull/1040/files#diff-ccb16d8308d46464c96de7ce7571ef9bL122). What types can this have ?

Reply:
I would expect `string|null`, but really forms can pass anything to a validator when a data transformer is added... That's why I annotated it `mixed`

Reply:
I've resolved the conflict using a merge commit (I plan to merge this PR using a squash and merge)